plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/utils/RemoteEnvUtils.kt

```diff
@@ -21,3 +24,8 @@ fun disableExtensionIfRemoteBackend() {
         throw ExtensionNotApplicableException.create()
     }
 }
+
+// CW can be supported only after at least build 232.9921.47 on remote env
+fun isRunningOnCWNotSupportedRemoteBackend() =
+    ApplicationInfo.getInstance().build.compareTo(BuildNumber.fromStringOrNull("232.9921.47")) < 0 &&
+        AppMode.isRemoteDevHost()
```

Comment:
can you change `isRunningOnRemoteBackend` to use `AppMode.isRemoteDevHost()`

Reply:
sure